README.md

```diff
@@ -160,7 +159,7 @@ Requirements:
 - At least one page in CMS (see above).
 - At least [15% of free disk space](https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html).
 - For Mac/Windows
-    - At least 4GB of RAM allocated to Docker (see Docker Desktop > Preferences > Resources > Advanced).
+    - At least 4GB of RAM allocated to Docker (see Docker Desktop > Settings > Resources > Advanced).
```

Comment:
I think macOS and Docker Desktop updated this. I have the latest version of both, as of this commit.

<details>

- The "Preferences…" option is not in the Docker Desktop menu.
- The "Preferences…" option is replaced with "Settings…" in other macOS apps.
- The "Settings" in Docker Desktop are accessible via a gear icon.
- Hover over gear icon to see tooltip "Settings".

</details>